rq/worker.py

```diff
@@ -157,7 +157,7 @@ def find_by_key(cls, worker_key, connection=None, job_class=None,
 
     def __init__(self, queues, name=None, default_result_ttl=None, connection=None,
                  exc_handler=None, exception_handlers=None, default_worker_ttl=None,
-                 job_class=None, queue_class=None):  # noqa
+                 job_class=None, queue_class=None, default_job_monitoring_interval=None):  # noqa
```

Comment:
One last nitpick (hopefully ;) ). Please change this to `job_monitoring_interval= DEFAULT_JOB_MONITORING_INTERVAL` .

`default_result_ttl` is prefixed with `default_` because it's used as default if jobs don't specify their own `result_ttl`.

Reply:
Shouldn't this be handled by https://github.com/rq/rq/pull/945/files/7ca105a2d5cab1551e7ba76863d173591a2db416#diff-1aac421e8c53e895ee40f3b7dc63144fR186 ?
I followed the same "style" as for `default_worker_ttl`...

Reply:
Sorry, the commit got rewritten - and the diff lost. Here is a working link: https://github.com/rq/rq/pull/945/files#diff-1aac421e8c53e895ee40f3b7dc63144fR186

Reply:
Feel free to change that "style" as well :). Default argument should be specified directly ini `__init__`.